SPECS/shadow-utils/shadow-utils.spec

```diff
@@ -150,6 +150,7 @@ find %{buildroot} -type f -name "*.la" -delete -print
 %post
 %{_sbindir}/pwconv
 %{_sbindir}/grpconv
+chmod 000 etc/shadow
```

Comment:
I feel like we should be managing this file through RPM, instead of leaving it as an untracked artifact of the `%post` scriptlet. We could get the same effect by adding the following line to the `%files` section:

```
%attr(0000,root,root) %config(noreplace,missingok) /etc/shadow
```

The `%attr` attribute sets the permissions to `0000`, the `%config(noreplace)` attribute ensures the file isn't messed with by any RPM upgrades , and the `%config(missingok)` attribute means RPM will track the file, but not expect it to be present at RPM creation (perfect for generating the file during the %post section).

Reply:
Redhat does some interesting stuff here: https://pagure.io/setup/blob/master/f/setup.spec#_80 but they have something very similar.

Reply:
add %attr(0000,root,root) %config(noreplace,missingok) %ghost /etc/shadow in &file section to make it trackable.